Projects/TEAM-2/Tsvihun/Task_221/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿// 221
+// Написати overload методи RemoveFromString, які приймають текст і видаляють з нього або певний набір символів, або символ\символи за порядковим номером в тексті.
+// Реалізувати використання аргументів за замовчуванням та статичних полів\методів
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Task_221
+{
+    
+    class Program
+    {
+        static string text = "Antonio and other members want to do some work together";
+        static string redudant_text = "aAr";
+
+
+        // Method removes from string certain char set (redudant_text)
+        static string remove_text(string input_text, string text_toremove = "e")
+        {
+            string output_text = "";
+            List<char> charlist = input_text.ToList();
+            var outputlist = charlist.Except(text_toremove);
+
+            foreach (char c in outputlist)
+            {
+                output_text += c;
+            }
+
+            return output_text;
+        }
+
+
+        // Method removes from string a char by index
+        static string remove_text(string input_text, int index_toremove)
+        {
+            string output_text = "";
+
+            List<char> charlist = input_text.ToList();
+            charlist.Remove(input_text[index_toremove]);
+
+            foreach (char c in charlist)
```

Comment:
I believe there should be an easier way to convert charList to string without loops
 https://stackoverflow.com/questions/58524222/c-converting-list-of-chars-to-string